frontend/packages/pipelines-plugin/src/components/repository/sections/WebhookSection.tsx

```diff
@@ -174,7 +178,7 @@ const WebhookSection: React.FC<WebhoookSectionProps> = ({ pac }) => {
                 showBadge
                 onChange={(k, v, res) => {
                   if (res && res.data) {
-                    setFieldValue('webhook.secretObj', res);
+                    setFieldValue(`${fieldPrefix}webhook.secretObj`, res);
```

Comment:
@Lucifergene why `fieldPrefix` is not being used here https://github.com/openshift/console/blob/c0d44afdfeaa073374e94eaf94c0a073b8378609/frontend/packages/pipelines-plugin/src/components/repository/sections/WebhookSection.tsx#L172?

Reply:
The feature was working without changing this value, therefore left it as it is. 
Now, have updated it